src/fund/hub/Spoke.sol

```diff
@@ -41,6 +45,15 @@ contract Spoke is DSAuth {
         setOwner(address(0));
     }
 
-    function priceSource() public view returns (address) { return hub.priceSource(); }
+    function priceSource() public view override returns (address) { return hub.priceSource(); }
     function fundFactory() public view returns (address) { return routes.fundFactory; }
+    function getHub() public view override returns (IHub) { return IHub(address(hub)); }
+    function getRoutes()
```

Comment:
Is this necessary because we can't override a public struct storage var?

Reply:
Exactly, though I think this will be [fixed in an upcoming version of solidity](https://github.com/ethereum/solidity/issues/8489)